src/DeviceInterface.js

```diff
@@ -175,6 +175,18 @@ class AppleDeviceInterface extends InterfacePrototype {
             notifyWebApp({isApp})
         }
 
+        this.setupAutofill = ({shouldLog} = {shouldLog: false}) => {
```

Comment:
This is the new setup flow for Apple Devices, where we brought back `isDeviceSignedIn`.